app/models/katello/ostree_branch.rb

```diff
@@ -10,7 +10,6 @@ class OstreeBranch < Katello::Model
     scoped_search :on => :commit, :complete_value => true
     scoped_search :on => :uuid, :complete_value => true
     scoped_search :on => :version_date, :complete_value => true, :rename => :created
-    scoped_search :relation => :repositories, :on => :name, :rename => :repository, :complete_value => true
```

Comment:
do this for modules also -> https://github.com/Katello/katello/blob/master/app/models/katello/module_stream.rb#L17